### ISSUE TITLE
voice-example Error WebSocketException

### ISSUE THREAD

**Issue by AxelRocha** [l7ssha/nyxx#56]
I tried to run voice-example and I'm getting this error:

> WebSocketException (WebSocketException: SocketException: OS Error: remote host refused connection.
> , errno = 1225, address = 127.0.0.1, port = 60609)

Trying to debug, I've noticed it begins [here](https://github.com/l7ssha/nyxx/blob/development/example/voice-example.dart#L15) and goes all the way to [here](https://github.com/l7ssha/nyxx/blob/development/lib/src/lavalink/LavalinkManager.dart#L81). When it tries to do transport.WebSocket.connect(...) the error happens.

Can u guys help?

Thanks in advance.

**Comment by l7ssha** [comment]
Do you have running Lavalink? Haven't tried it for a while, compatibility may be lacking due I didnt updated it

**Comment by AxelRocha** [comment]
I didnt knew I need it.

But then, I searched for it and found https://github.com/Frederikam/Lavalink. I followed its instructions [Server Configuration](https://github.com/Frederikam/Lavalink#server-configuration), and has able to run the Lavalink.

I tried to run the voice-example again and the same error appers. 

> Exception has occurred.
> WebSocketException (WebSocketException: SocketException: OS Error: remote host refused connection.
> , errno = 1225, address = 127.0.0.1, port = 51360)

(almost the same error, errno is the same, but the port has changed)

Any thoughts?

Is anything I need to do and/or prepare in my environment to run the voice-exemple?
